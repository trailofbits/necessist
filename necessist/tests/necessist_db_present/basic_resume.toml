args = ["--root=examples/basic", "--resume"]
stderr = ""

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 0
