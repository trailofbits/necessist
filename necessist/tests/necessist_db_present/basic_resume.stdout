4 candidates in 4 tests in 1 test file
